lib/main.dart

```diff
@@ -1,23 +1,35 @@
 import 'package:flutter/material.dart';
-// import 'package:langx_flutter/pages/home/bottom_navigation_bar.dart';
+import 'package:appwrite/appwrite.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
 import 'package:langx_flutter/login.dart';
-// import 'package:langx_flutter/old/community.dart';
-// import 'package:langx_flutter/pages/home/profile.dart';
-// import 'package:langx_flutter/pages/home/profile.dart';
-// import 'package:langx_flutter/pages/home/home.dart';
 
-void main() {
-  runApp(const Main());
+Future<void> main() async {
```

Comment:
Let's keep this as void main()
And better add the appwrite initialization
`  Client client = Client()
      .setEndpoint(dotenv.env['APP_ENDPOINT']!)
      .setProject(dotenv.env['APP_PROJECT']!);
  Account account = Account(client);`
  
  In a separate file in order to reference it in different places easier

Reply:
i dont know how can i do, can you please also attach a new commit to this PR to do that?